src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/OpenAPIConfiguration.java

```diff
@@ -14,12 +17,14 @@
 public class OpenAPIConfiguration {
 
     @Bean
-    public GroupedOpenApi publicApi(OperationCustomizer customGlobalHeaders) {
-        return GroupedOpenApi.builder()
-            .group("case-event-handler-public")
-            .pathsToMatch("/**")
-            .addOperationCustomizer(customGlobalHeaders)
-            .build();
+    public OpenAPI openAPI() {
+        return new OpenAPI()
+            .info(new Info().title("wa-case-event-handler Service")
+                      .version("v1.0.0")
+                      .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
+            .externalDocs(new ExternalDocumentation()
+                              .description("README")
+                              .url("https://github.com/hmcts/wa-case-event-handler"));
```

Comment:
formatting correct here?

Reply:
I made these changes identical to this file Ravi.

Reply:
It's only the formatting that seems incorrect. Looks like double tabs rather than single?  Nothing major.